gapic/utils/lines.py

```diff
@@ -56,16 +56,18 @@ def get_subsequent_line_indentation_level(list_item: str) -> int:
     + The quick brown fox jumps over the lazy dog. The quick brown fox jumps
       over the lazy dog
 
-    Here subsequent lines should be indented by 3
+    Here subsequent lines should be indented by 4 to cater for double digits
 
-    1. The quick brown fox jumps over the lazy dog. The quick brown fox jumps
-       over the lazy dog
+    1.  The quick brown fox jumps over the lazy dog. The quick brown fox jumps
+        over the lazy dog
 
+    22. The quick brown fox jumps over the lazy dog. The quick brown fox jumps
+        over the lazy dog
     """
     if len(list_item) >= 2 and list_item[0:2] in ['- ', '+ ']:
         indentation_level = 2
-    elif len(list_item) >= 3 and list_item[0].isdigit() and list_item[1:3] == '. ':
-        indentation_level = 3
+    elif len(list_item) >= 4 and re.match("^\d+\. ", list_item):
```

Comment:
Considering factoring out the regex itself into a constant, so we have a single source of truth, rather than repeating it here and below. (But granted, it's not terribly complicated.)

Reply:
Fixed in [73dd48f](https://github.com/googleapis/gapic-generator-python/pull/1740/commits/73dd48f25319e085fc878409289b0073fcbadf47)